workflow/runtime/suite.yaml

```diff
@@ -2,6 +2,9 @@
 # suites in the workflow/ directory.  Most of these variables are
 # specific to the cycled workflow which is not in this release.
 
+day_dt: !timedelta "24:00:00"
+zero_dt: !timedelta "00:00:00"
```

Comment:
I think this answers my question earlier...these are constants (0 and 24) that you use elsewhere?

Reply:
Yes